charts/m4d/templates/webhook-certificates.yaml

```diff
@@ -1,5 +1,5 @@
 {{- if include "m4d.isEnabled" (tuple .Values.manager.enabled (or .Values.coordinator.enabled .Values.worker.enabled)) }}
-apiVersion: cert-manager.io/v1alpha2
+apiVersion: cert-manager.io/v1
```

Comment:
I suggest to revert this change. In face, we have an open issue to support back to 0.10.x https://github.com/IBM/the-mesh-for-data/issues/581